.circleci/config.yml

```
@@ -105,22 +150,139 @@ jobs:
                 name: Upload code coverage
                 command: cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
 
+  release_prepare:
+    machine: true
+    steps:
+      - checkout
+      - bootstrap_repository_command
+      - run:
+          name: Check if packages are ready to be released
+          command: yarn run release:prepare-packages --verbose --compile-only
```

To be consistent with other `release:*` scripts:
```suggestion
          command: npm run release:prepare-packages -- --verbose --compile-only
```

---

91771e4